tests/queries/0_stateless/01442_merge_detach_attach_long.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-# Tags: long, no-parallel, no-debug
```

Comment:
Why run it in debug? (also applies to the previous one)

Reply:
It's the version I was using locally to test them. I'll add it back